sbt_common/messaging/src/test/scala/uk/ac/wellcome/messaging/test/fixtures/Messaging.scala

```diff
@@ -112,25 +112,25 @@ trait Messaging
     testWith(stream)
   }
 
-  def withMessageStreamFixtures[R](
-    testWith: TestWith[(Bucket,
-    MessageStream[ExampleObject],
+  def withMessageStreamFixtures[T,R](
+    testWith: TestWith[(ActorSystem, Bucket,
```

Comment:
Why are we adding the ActorSystem to the returned outputs of the fixture? I can’t see it being used anywhere.

Reply:
ah right, it was used at some point, but it's not anymore. Thanks, I'll remove that

Reply:
Cool. Happy for you to merge once that’s done, I don’t need another review.